docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.9"
```

Comment:
Still feel shaky on having the group install docker.  Is it possible to reserve docker for exclusively tests that absolutely need it and still have others continue to use the H2 db?  Essentially the goal being that most of the group can run and write tests without needing docker installed.  That way only a couple members need docker specifically for the tests that require docker.